recipes/simplemseed/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "simplemseed" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "7c268a35441d7e85d8078ee57b9c20cf2b9fcf23481a9c3548565ffbf37cd401" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

Please use grayskull to generate the recipe. It will make it much easier to review/merge:

https://github.com/conda-forge/staged-recipes/?tab=readme-ov-file#grayskull---recipe-generator-for-python-packages-on-pypi

```suggestion

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: 7c268a35441d7e85d8078ee57b9c20cf2b9fcf23481a9c3548565ffbf37cd401
```

---

I used grayskull but moved the hash to the top, since it just makes more sense to me to have everything that changes from version to version together in one place at the top. But I can change it :+1:

---

The bot will update that for you. We try to avoid excess of jinja to make a future recipe 2.0 format migration easier. (All that jinja magic will probably go away in liue for a more declarative and clear recipe.)